packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -723,7 +723,7 @@ export default [
                     redirectOnSuccess='/backup-ng'
                     size='large'
                   >
-                    {_('scheduleEdit')}
+                    {_('saveBackupJob')}
```

Comment:
Create new generic entries in `messages`:
- `formCreate`: when the form leads to the creation of a new entity
- ` formSave`: when the form leads to the modification of an entity
- `formCancel`: close the form without side-effect
- `formReset`: reset the state of a form